### PR TITLE
Initializing joining channels

### PR DESCRIPTION
**Pull request by overlordbleck**
My working PR for joining channels in response to #10

### CODE REVIEW

**Comment by overlordbleck** [comment]
Oof I made a mistake, going in and fixing it

**Comment by overlordbleck** [comment]
Works flawlessly now! =^)

File: lua/cfc_voice_channels/core/sv/sv_channel_control.lua [hunk 1]
```diff
@@ -53,6 +57,7 @@ function cfc_voice:CreateChannel(caller, name, password)
 
     cfc_voice.Channels[table.Count(cfc_voice.Channels) + 1] = {
         ["Name"] = channelName,
+        ["TrimedName"] = string.lower(string.Trim(channelName)),
```
Comment (sarahsturgeon) [inline lua/cfc_voice_channels/core/sv/sv_channel_control.lua]: Trimmed*

File: lua/cfc_voice_channels/core/sv/sv_channel_control.lua [hunk 2]
```diff
@@ -66,14 +71,34 @@ end
...
 end
 
+function cfc_voice:getChannel(channelName)
+    for _, channel in pairs(self.Channels) do
+        if channel.TrimedName == string.lower(string.Trim(channelName)) then
+            return channel
+        end
+    end
+
+    return nil
```
Comment (sarahsturgeon) [inline lua/cfc_voice_channels/core/sv/sv_channel_control.lua]: No need for a return here

**Review by sarahsturgeon — CHANGES_REQUESTED** [review]
quick feedback, will look at it more later

File: lua/cfc_voice_channels/core/sv/sv_channel_control.lua (hunk 1 above)
Comment (overlordbleck) [reply lua/cfc_voice_channels/core/sv/sv_channel_control.lua]: Oof can’t spell, I’ll fix when I get home lol...

File: lua/cfc_voice_channels/core/sv/sv_channel_control.lua [hunk 3]
```diff
@@ -66,14 +71,34 @@ end
 
 function cfc_voice:isUniqueChannelName(name)
     for _, channel in pairs(cfc_voice.Channels) do
-        if string.lower(string.Trim(channel.Name)) == string.lower(string.Trim(name)) then
+        if channel.TrimedName == string.lower(string.Trim(name)) then
```
Comment (ajs5710) [inline lua/cfc_voice_channels/core/sv/sv_channel_control.lua]: Here too

File: lua/cfc_voice_channels/core/sv/sv_channel_control.lua [hunk 4]
```diff
@@ -66,14 +71,34 @@ end
...
             return false
         end
     end
 
     return true
 end
 
+function cfc_voice:getChannel(channelName)
+    for _, channel in pairs(self.Channels) do
+        if channel.TrimedName == string.lower(string.Trim(channelName)) then
```
Comment (ajs5710) [inline lua/cfc_voice_channels/core/sv/sv_channel_control.lua]: At least your spelling is consistent

Comment (overlordbleck) [reply lua/cfc_voice_channels/core/sv/sv_channel_control.lua]: Copy and paste is a bitch

**Review by sarahsturgeon — APPROVED** [review]
LGTM as long as you're planning on doing a final cleanup sweep

**Pull request merged**